hikari/api/rest.py

```diff
@@ -1183,7 +1675,28 @@ async def edit_my_user(
         username: undefined.UndefinedOr[str] = undefined.UNDEFINED,
         avatar: undefined.UndefinedNoneOr[files.Resourceish] = undefined.UNDEFINED,
     ) -> users.OwnUser:
-        ...
+        """Edit the token associated user.
```

Comment:
Was wondering if this (and line 1656) would read better in a possessive form (the token's associated).